pkg/network/ovn_kubernetes_test.go

```diff
@@ -910,6 +912,55 @@ logfile-maxage=0`,
 			controlPlaneReplicaCount: 2,
 			enabledFeatureGates:      []configv1.FeatureGateName{apifeatures.FeatureGateDNSNameResolver},
 		},
+		{
+			desc: "enable PreconfiguredUDNAddresses, UDN, and static-IPs",
+			expected: `
+[default]
+mtu="1500"
+cluster-subnets="10.128.0.0/15/23,10.0.0.0/14/24"
+encap-port="8061"
+enable-lflow-cache=true
+lflow-cache-limit-kb=1048576
+enable-udp-aggregation=true
+udn-allowed-default-services="default/kubernetes,openshift-dns/dns-default"
+
+[kubernetes]
+service-cidrs="172.30.0.0/16"
+ovn-config-namespace="openshift-ovn-kubernetes"
+apiserver="https://testing.test:8443"
+host-network-namespace="openshift-host-network"
+platform-type="GCP"
+healthz-bind-address="0.0.0.0:10256"
+dns-service-namespace="openshift-dns"
+dns-service-name="dns-default"
+
+[ovnkubernetesfeature]
+enable-egress-ip=true
+enable-egress-firewall=true
+enable-egress-qos=true
+enable-egress-service=true
+egressip-node-healthcheck-port=9107
+enable-multi-network=true
+enable-network-segmentation=true
+enable-preconfigured-udn-addresses=true
+enable-multi-external-gateway=true
+
+[gateway]
+mode=shared
+nodeport=true
+
+[logging]
+libovsdblogfile=/var/log/ovnkube/libovsdb.log
+logfile-maxsize=100
+logfile-maxbackups=5
+logfile-maxage=0`,
+			controlPlaneReplicaCount: 2,
+			enabledFeatureGates: []configv1.FeatureGateName{
+				apifeatures.FeatureGateNetworkSegmentation,
+				apifeatures.FeatureGatePreconfiguredUDNAddresses,
+				apifeatures.FeatureGatePersistentIPsForVirtualization,
```

Comment:
good catch from @Meina-rh given this feature gate was already removed via https://github.com/openshift/cluster-network-operator/pull/2737 we shouldn't have added this back.. its gonna cause unit test failures